impl/src/bin/cargo-raze.rs

```diff
@@ -144,16 +146,18 @@ fn main() -> Result<()> {
     if dry_run {
       println!("{}:\n{}", path.display(), contents);
     } else {
-      write_to_file_loudly(&path, &contents)?;
+      write_to_file(&path, &contents, options.flag_verbose > 0)?;
     }
   }
 
   Ok(())
 }
 
-fn write_to_file_loudly(path: &Path, contents: &str) -> Result<()> {
+fn write_to_file(path: &Path, contents: &str, verbose: bool) -> Result<()> {
```

Comment:
This should probably be a separate PR or at least mentioned in the PR description.

Reply:
Which would you prefer? I can do either

Reply:
I made #263